src/codegen/mod.rs

```diff
@@ -2198,7 +2198,13 @@ impl CodeGenerator for CompInfo {
             let prefix = ctx.trait_prefix();
             result.push(quote! {
                 impl #generics Default for #ty_for_impl {
-                    fn default() -> Self { unsafe { ::#prefix::mem::zeroed() } }
+                    fn default() -> Self {
```

Comment:
Might be worth a comment before `result.push(...)` mentioning why we're using write_bytes etc rather than `zeroed()`.